bin/tictactoe

```diff
@@ -1,3 +1,35 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+puts "Welcome to Tic Tac!"
+puts "Select number of human players: (0-2)"
+  input = gets.strip
+  if input == "2"
+    puts "First player make your move!"
+    g = Game.new
+    g.play
+    g.exit
+  elsif input == "1"
+    puts "Who will go first? \n Choose 1 for Computer \n Choose 2 for Human"
+    first_player = gets.strip
+    if first_player == "1"
+      g = Game.new(Computer.new("X", true))
+      g.play
+      g.exit
+    elsif first_player == "2"
+      g = Game.new(Human.new("X"), Computer.new("O", true))
+      g.play
+      g.exit
+    end    
+  elsif input == "0"
+    g = Game.new(Computer.new("X", true), Computer.new("O", true))
+    g.play
+    g.exit
+  else
+    puts "Try choosing from the options next time."
+    Game.new.exit
+  end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.